ch_pipeline/core/telescope.py

```diff
@@ -532,19 +534,25 @@ class CHIMEExternalBeam(CHIME):
     This class uses an external beam model that is read in from a file.
     """
 
-    primary_beamx_filename = config.Property(
+    primary_beam_filename = config.Property(
         proptype=str,
         default="/project/k/krs/cahofer/pass1/beams/beamx_400_800_nfreq200.hdf5",
```

Comment:
this file doesn't exist anymore, we should link to the latest SVD primary beam model for example

Reply:
I guess it doesn't need a default value at all.